src/main/java/com/example/generativeai/entity/AuthorizedUserEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.example.generativeai.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Entity representing authorized user metadata.
+ */
+@Data
```

Comment:
Don't use `@Data` on _Entities_.
https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/

Reply:
Thanks for the link. Fixed